units/commit_queue.go

```diff
@@ -168,6 +176,13 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 	if !valid {
 		return
 	}
+	conf := j.env.Settings()
+	githubToken, err := conf.GetGithubOauthToken()
+	if err != nil {
+		j.AddError(errors.Wrap(err, "can't get github token"))
+		j.AddError(errors.Wrap(cq.SetProcessing(false), "can't set processing to false"))
```

Comment:
You don't have to set processing to false since you're getting this above line 201 now.